source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +51,65 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
-Visualize the alerts
---------------------
+Visualizing the alerts
```

Comment:
This section appears outside the vulnerability detection main section in the left panel 
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/00a46efa-2a23-4ec4-8a05-56bb7c4fc695)

Reply:
Done!